docs/usage/editing-paths.md

```diff
@@ -0,0 +1,78 @@
+| --------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------- | ------------- |
+| Waypoint Velocity Direction | Constrains the robot's direction of travel at a waypoint.                                                                                                                                                                                            | Yes            | No            |
+| Waypoint Zero Velocity      | Stop at a certain point                                                                                                                                                                                                                              | Yes            | No            |
+| Max Velocity                | Constrains the max velocity to a specific value throughout multiple waypoints/segments This generally refers to the Maximum Velocity of robot chassis in m/s. A default value can be defined in Robot Config ([Getting Started](../robot-config.md)) | Yes            | Yes           |
+| Zero Angular Velocity       | Ensures zero angular velocity through the waypoint and/or segment scope.                                                                                                                                                                             | Yes            | Yes           |
+| Straight Line               | Tells the solver to draw a straight line as the path between two immediate waypoints.                                                                                                                                                                | No             | Yes           |
+
+## Obstacles
+
+Obstacles allow the robot to avoid a certain area of the field throughout the path. For example, in the 2023 Charged Up game, you might want to avoid crashing into the charge station. Obstacles makes it to define where you want the robot to start and end without unnecessary intermediary waypoints.
+
+To start, select the circle obstacle from the navbar (looks like a circle with a slash through it).
+
+Then, select anywhere on the field to add an obstacle. You can click on the ends of the circle to resize them.
+
+![Obstacles](../media/obstacles.png)
+
+## Generating
+
+You can create paths by chaining waypoints together. Check out [Controls & Shortcuts](./controls-shortcuts.md) for advanced controls. Once you have at least two waypoints, then you can generate a path. You can cancel the path generation by pressing control + clicking the red X once it has started.
+
+If there is an error generating a path, check to make sure your waypoints can physically work.
```

Comment:
Would probably be better as an [admonition](https://squidfunk.github.io/mkdocs-material/reference/admonitions/).